sdk/internal/recording/recording.go

```diff
@@ -585,11 +588,30 @@ func Start(t *testing.T, pathToRecordings string, options *RecordingOptions) err
 		}
 		return fmt.Errorf("Recording ID was not returned by the response. Response body: %s", b)
 	}
+
+	// Unmarshal any variables returned by the proxy
+	var m map[string]interface{}
+	body, err := ioutil.ReadAll(resp.Body)
```

Comment:
Looks like `resp.Body` isn't being closed.  We always need to call `resp.Body.Close()`.